### ISSUE TITLE
IconService call method fails with exception "[NETWORK ERROR] fetch is not defined"

### ISSUE THREAD

**Issue by 0xrpj** [icon-project/icon-sdk-js#60]
Hi.
I am trying to implement icon-sdk in a typescript project.

```
  public async invokeCallBuilder(score: string, method: string, params: any): Promise<any> {
    const call = new CallBuilder().to(score).method(method).params(params).build();
    try {
      const response = await iconService.call(call).execute();
      // The above line returns fetch is not defined 
      // Code doesnot reach here!
      return response;
    } catch (error) {
      console.log('error: ', error);
    }
  }

```

This is my implementation. The values passed here are fine. 
Printing out the call variable returns this
`p { provider: { url: 'https://berlin.net.solidwallet.io/api/v3' } }`
and this looks fine.

But the code immediately fails when it reaches the iconService.call(call) part. 
Here, is how I have initialized iconService:

```
import Icon from 'icon-sdk-js';
const { IconWallet, IconAmount, IconConverter, IconBuilder, SignedTransaction, HttpProvider } = Icon;
const { CallBuilder, CallTransactionBuilder } = IconBuilder;
const provider = new HttpProvider(process.env.NETWORK_URL);
const iconService = new Icon(provider);
```

Here is the error received:
![image](https://user-images.githubusercontent.com/56220537/212237723-2fc5306c-bcae-48a7-821b-9914c79e222a.png)

**Comment by 0xrpj** [comment]
icon-sdk-js version is ^1.2.12

**Comment by inwonkim** [comment]
@0xrpj Are you using icon-sdk in a environment with fetch support? If not, you should polyfil it.

**Comment by 0xrpj** [comment]
I am using it in a project with nodejs. @inwonkim

**Comment by 0xrpj** [comment]
This issue is fixed by downgrading to this version: 1.2.1!
